recipes/mpir/bld.bat

```diff
@@ -0,0 +1,19 @@
+cd build.vc14
```

Comment:
See https://github.com/conda/conda-build/wiki/Windows-recipe-patterns#version-specific-solution-files

Cut out the cmake stuff, keep the VSTRING stuff.  Keep the trim part if you find you get whitespace and that trips you up.  VSTRING will be something like "9" or "10" or "14"